pass-bakery-fe/src/components/BakeryMain.jsx

```diff
@@ -0,0 +1,19 @@
+import { menuData } from "../data/menuData";
+import TitleNav from "./TitleNav";
+import Menu from "./Menu";
+
+function BakeryMain(props) {
+  const menuObj = {};
+  menuData.menu.map((item, idx) => {
+    return (menuObj[menuData.menu[idx].name] = item);
```

Comment:
you don't need to return the setter here.

Reply:
I just added it in because eslint kept giving me a warning for it, will remove.